backend/database/models/suspect.py

```diff
@@ -0,0 +1,22 @@
+from backend.database.models._assoc_tables import suspect_officer
+from backend.database.models.officer import Rank
+from .. import db
+
+
+class Suspect(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    incident_id = db.Column(db.Integer, db.ForeignKey("incident.id"))
+    first_name = db.Column(db.Text)
+    last_name = db.Column(db.Text)
+    race = db.Column(db.Text)
+    gender = db.Column(db.Text)
+    badge = db.Column(db.Text)
+    unit = db.Column(db.Text)  # type?
+    # Note: rank at time of incident
+    rank = db.Column(db.Enum(Rank))
+    star = db.Column(db.Text)  # type?
+    suspected_matches = db.relationship(
+        "Officer", secondary=suspect_officer, backref="suspect_matches")
```

Comment:
The inconsistency between `suspected_matches` and `suspect_matches` has me wondering if this is 1-to-1 where there is an officer that is definitely the suspect or if it's 1-to-many where there are officers who are suspected of being the suspect?

Reply:
Definitely a many to many. Suspect may be disproven later and that would be recorded in this relationship. Also, you'd need to be able to have a new one.

The naming is clunky though; I agree with you.

Reply:
Changing this to `Perpetrator` per slack discussion.